packages/tasks/src/const.ts

```diff
@@ -18,6 +18,7 @@ export const TASKS_MODEL_LIBRARIES: Record<PipelineType, ModelLibraryKey[]> = {
 	"image-segmentation": ["transformers", "transformers.js"],
 	"image-to-image": [],
 	"image-to-text": ["transformers.js"],
+	"image-to-video": [], // TODO: diffusers will soon support image-to-video
```

Comment:
Let's add it already, it will be ready this week

```suggestion
	"image-to-video": ["diffusers"],
```

Reply:
@osanseviero shouldn't this line also be diffusers ? https://github.com/huggingface/huggingface.js/blob/6764b0e2e256fd79cd718ec6d90fbcccedecea9c/packages/tasks/src/const.ts#L39

Reply:
wdyt e1d2b92c9207e7162f423169b4af437966a80ae5 ?

Reply:
yep TBH i didn't even remember the tasks/src/const.ts one

Reply:
Yes @mishig25 but let's keep this PR for image-to-video? For text-to-image actually I have https://github.com/huggingface/huggingface.js/pull/320 being updated with the migration

Reply:
They are different though. `LibrariesToTasks` defines which tasks are supported by the Inference API for the specific library. This file is automatically generated by a GitHub Actions workflow in `api-inference-community`. `const.ts` is about library support, not specific to Inference API

Reply:
handled in cb0f27702879889c1e18102113a9ea6764946f5c and 6764b0e2e256fd79cd718ec6d90fbcccedecea9c